web/website/content/_index.md

```diff
@@ -169,8 +175,10 @@ showcase_section:
         SELECT
           *,
           started + unfinished AS finished,
-          ((started + unfinished) * 1.0 / started) AS fin_share,
-          ( ((started + unfinished) * 1.0 / started) * 1.0 / (1 - ((started + unfinished) * 1.0 / started)) ) AS fin_ratio
```

Comment:
I replaced this with the current results in e355722 because I was concerned about the space.